.discourse-compatibility

```diff
@@ -0,0 +1 @@
+3.1.0.beta1: 96211d622e15793574d94633f5ce4403f36b65ff
```

Comment:
Are we sure that .discourse-compatibility works correctly here, given that docker_manager is the thing actually performing the upgrade. I worry it might cause docker_manager to continuously have a 'pending' upgrade?